src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedFieldFactory.test.js

```diff
@@ -2,72 +2,120 @@ import '@folio/stripes-acq-components/test/jest/__mock__';
 
 import { FixedFieldFactory } from './FixedFieldFactory';
 
-import BookFixedField from './BookFixedField';
-import ComputerFileFixedField from './ComputerFileFixedField';
-import ContinuingResourceFixedField from './ContinuingResourceFixedField';
-import MapFixedField from './MapFixedField';
-import MixedMaterialFixedField from './MixedMaterialFixedField';
-import ScoreFixedField from './ScoreFixedField';
-import SoundRecordingFixedField from './SoundRecordingFixedField';
-import VisualMaterialFixedField from './VisualMaterialFixedField';
+import BibliographicFixedField from './BibliographicFixedField';
 import HoldingsFixedField from './HoldingsFixedField';
 import AuthorityFixedField from './AuthorityFixedField';
 import { MARC_TYPES } from '../../../common/constants';
+import marcSpecificationBib from '../../../../test/mocks/marcSpecificationBib';
+import marcSpecificationAuth from '../../../../test/mocks/marcSpecificationAuth';
+import marcSpecificationHold from '../../../../test/mocks/marcSpecificationHold';
 
 describe('FixedFieldFactory', () => {
-  it('should create correct fields', () => {
+  it('should create correct marc type fixed field', () => {
     expect(
-      FixedFieldFactory.getFixedField('records', MARC_TYPES.BIB, 'a', 'm').type.displayName,
-    ).toBe(BookFixedField.displayName);
+      FixedFieldFactory.getFixedField('records', MARC_TYPES.BIB).type.displayName,
+    ).toBe(BibliographicFixedField.displayName);
 
     expect(
-      FixedFieldFactory.getFixedField('records', MARC_TYPES.BIB, 't').type.displayName,
-    ).toBe(BookFixedField.displayName);
+      FixedFieldFactory.getFixedField('records', MARC_TYPES.AUTHORITY).type.displayName,
+    ).toBe(AuthorityFixedField.displayName);
 
     expect(
-      FixedFieldFactory.getFixedField('records', MARC_TYPES.HOLDINGS, 't').type.displayName,
+      FixedFieldFactory.getFixedField('records', MARC_TYPES.HOLDINGS).type.displayName,
     ).toBe(HoldingsFixedField.displayName);
+  });
 
+  it('should create correct document type', () => {
```

Comment:
Also I think `fixed field type` would be better name than `document type`

Reply:
ok